README.md

```diff
@@ -125,6 +125,7 @@ Aside from those listed here, many other apps and libraries can be easily be fou
 - [Thardians](https://gitlab.com/thustle/thardians-rs) - Space Invaders for the terminal.
 - [tic-tac-toe](https://github.com/thomas-mauran/tic-tac-toe) - Terminal-based tic tac toe game.
 - [wordl](https://github.com/palerdot/wordl-rs) - Terminal-based Wordle game. Web like experience with keyboard hints and guess reveal animations.
+- [plastic](https://github.com/Amjad50/plastic) - NES emulator with extra ui implemented in ratatui.
```

Comment:
Can you place this accordingly to the alphabetical order? :)

Reply:
ah didn't notice, yes, I"ll fix it

Reply:
Fixed

Reply:
would be good to maybe have this as part of the linter CI